packages/lesswrong/components/posts/PostsList2.tsx

```diff
@@ -42,6 +55,7 @@ const PostsList2 = ({classes, ...props}: PostsList2Props) => {
     placeholderCount,
     showFinalBottomBorder,
     viewType,
+    showPlacement,
```

Comment:
It makes me sad to further customize this guy. I don't think this should block this deploy, but consider removing / refactoring after the event is done to make it less impactful on this component.

Reply:
Yeah I didn't really want to do this, but I think it turned out not that bad (since I didn't have to add any wrappers around the post item), and if Toby is happy with this event UI then he will prob want to show this again in the future